<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>686</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>450</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_2" >
        <property name="title" >
         <string>Instructions</string>
        </property>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="informationLabel" >
           <property name="text" >
            <string/>
           </property>
           <property name="wordWrap" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox" >
        <property name="title" >
         <string>Folders</string>
        </property>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Select which folders to fingerprint.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CheckDirTree" name="dirTree" />
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>0</number>
      </property>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="fingerprintButton" >
        <property name="minimumSize" >
         <size>
          <width>88</width>
          <height>28</height>
         </size>
        </property>
        <property name="text" >
         <string>Fingerprint</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>548</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAccount" >
    <property name="title" >
     <string>&amp;Account</string>
    </property>
    <addaction name="actionLogout" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionFAQ" />
   </widget>
   <addaction name="menuAccount" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionLogout" >
   <property name="text" >
    <string>&amp;Log out</string>
   </property>
  </action>
  <action name="actionFAQ" >
   <property name="text" >
    <string>&amp;FAQ</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckDirTree</class>
   <extends>QTreeView</extends>
   <header>CheckDirTree.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
